packages/nova-forms/lib/Form.jsx

```diff
@@ -333,6 +336,22 @@ class Form extends Component {
       errors: _.uniq([...prevState.errors, error])
     }));
   }
+  
+  // add to cleared fields
+  clearField(fieldName){
+    this.setState({
+      deletedFields: [...this.deletedFields, fieldName]
+    });
+    
+    return () => this.removeFromDeletedfields(fieldName);
```

Comment:
Interesting pattern! However how would you trigger the returned function after someone fills again the component?

Btw, `removeFromDeletedfields` isn't defined 👌